### ISSUE TITLE
[iOS] xharness doesn't boot ios simulators

### ISSUE THREAD

**Issue by EgorBo** [dotnet/xharness#223]
None of the tests work for x64 in dotnet/runtime due to "Failed to find suitable device for target ios-simulator-64" in Xharness.

Xharness CLI: 1.0.0-prerelease.20271.3 (latest as of today).

Command:
```
XHarness command issued: ios test --app=/Users/egorbo/prj/runtime/artifacts/bin/System.Memory.Tests/net5.0-Release/ios-x64/AppBundle/System.Memory.Tests/Release-iphonesimulator/System.Memory.Tests.app --targets=ios-simulator-64 --output-directory=/Users/egorbo/prj/runtime/artifacts/bin/System.Memory.Tests/net5.0-Release/ios-x64/AppBundle/xharness-output
```
Output:
```
  info: test[0]
        Starting test for ios-simulator-64..
  info: test[0]
        Starting application 'System.Memory.Tests' on ios-simulator-64
  fail: test[0]
        Failed to find suitable device for target ios-simulator-64
  XHarness exit code: 1105
```

Log:
```
13:12:32.6523440 13:12:32.6426010 /usr/libexec/PlistBuddy -c "Print CFBundleName" /Users/egorbo/prj/runtime/artifacts/bin/System.Memory.Tests/net5.0-Release/ios-x64/AppBundle/System.Memory.Tests/Release-iphonesimulator/System.Memory.Tests.app/Info.plist
13:12:32.6665540 13:12:32.6665350 /usr/libexec/PlistBuddy -c "Print CFBundleIdentifier" /Users/egorbo/prj/runtime/artifacts/bin/System.Memory.Tests/net5.0-Release/ios-x64/AppBundle/System.Memory.Tests/Release-iphonesimulator/System.Memory.Tests.app/Info.plist
13:12:32.6728880 13:12:32.6728790 /usr/libexec/PlistBuddy -c "Print UIRequiredDeviceCapabilities" /Users/egorbo/prj/runtime/artifacts/bin/System.Memory.Tests/net5.0-Release/ios-x64/AppBundle/System.Memory.Tests/Release-iphonesimulator/System.Memory.Tests.app/Info.plist
13:12:32.6776540 13:12:32.6776350 Failed to get the UIRequiredDeviceCapabilities Info.plist property. Assumes 32 bit is not supported
13:12:32.7058900 13:12:32.7058800 Test log server listening on: 0.0.0.0:64596
13:12:34.0199380 13:12:34.0199250 /Applications/Xcode.app/Contents/Developer/usr/bin/simctl create "iPhone X (iOS-13-4) - created by xharness" com.apple.CoreSimulator.SimDeviceType.iPhone-X com.apple.CoreSimulator.SimRuntime.iOS-13-4
13:12:34.1062260 13:12:34.1062010 Invalid runtime: com.apple.CoreSimulator.SimRuntime.iOS-13-4
```

**Comment by premun** [comment]
@EgorBo please make sure Xcode 11.4 is installed on your system together with iOS 13.4 Simulator

**Comment by premun** [comment]
This was resolved by installation of iOS 13.4 Simulator. I have improved the errors to make it more clear that this is the cause
